### ISSUE TITLE
profiles() class loads Nprof > 1 profiles in wrong order

### ISSUE THREAD

**Issue by cgrdn** [ArgoCanada/BGC-QC#24]
When there is more than one profile in a single file (i..e `Nprof > 1`), there are `Nprof` number of timestamps. When using `profiles()` the multiple profiles seem to load in the wrong order. Example shown below: 

```python
>>> syn  = sprof(4902549)
>>> prof = profiles(4902549)
>>> syn.CYCLE
Out: array([1, 1, 2, 3, 4, 5])
>>> syn.SDN
Out: array([18491.57777778, 18493.38472222, 18503.37916667, 18513.36875   ,
       18523.375     , 18533.38055556])
>>> prof.CYCLE
Out: array([1, 1, 2, 3, 4, 5])
>>> prof.SDN
Out: array([18493.38472222, 18491.57777778, 18503.37916667, 18513.36875   ,
       18523.375     , 18533.38055556])
```

This is an issue because (1) I would like time to be monotonically increasing and (2) loading data via `profiles()` should get the same result as loading it with `sprof()`, right now that are not self-consistent.

**Comment by cgrdn** [comment]
Similar to my last time working with this, its not actually the case where Nprof > 1 is the issue I don't think. In this case, there are two files for cycle 1, which have different timestamps apparently: 

```python
>>> prof = profiles(4902549)
>>> prof.__argofiles__ # hidden variable with file list
Out: 
['/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_001.nc',
 '/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_001D.nc',
 '/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_002.nc',
 '/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_003.nc',
 '/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_004.nc',
 '/Users/gordonc/Documents/data/Argo/meds/4902551/profiles/BR4902551_005.nc']
```

I'm unclear what a_trailing_ D means, however that's not really important at this moment. Proposed fix is to organize files by time, which I'll use the global index for.

**Comment by cgrdn** [comment]
Fixed as suggested above in 78e3d3d
